proper_parenthetics.py

```diff
@@ -0,0 +1,27 @@
+"""This function takes a unicode string (text)
+as input and returns whether the string is properly closed
+parenthetically."""
+
+from doubly_linked_list import Doubly_Linked_List
+
+def proper_parenthetics(text):
+	"""Returns a 1 if the string is parenthetically open.
+	Returns a 0 if the string is parenthetically balanced.
+	Returns a -1 if the string is parenthetically broken."""
+	parenthetics = Doubly_Linked_List()
+	text = text[::-1]
+	for char in text:
+		if char == '(' or char == ')':
```

Comment:
`if char in ["(", ")"]`

Indentation here is waaaaaay too large. Are you using actual `tab` characters? Or are you using 4 spaces like you were supposed to.